### PR TITLE
update template README

### PR DESCRIPTION
**Pull request by hey-kong**

### CODE REVIEW

File: _template/README.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ github.com/kubeedge/mappers-go$ make template
 Please input the mapper name (like 'Bluetooth', 'BLE'): test
 ```
 A folder named as you input will be generated in the directory "mappers". The file tree is as below:
-mapper
+```
+mappers/test
```
Comment (RyanZhaoXB) [inline _template/README.md:14]: it seems that no need to change the root dirctory from "mapper" to "mappers/test".

Comment (hey-kong) [reply _template/README.md]: ok

**Review by RyanZhaoXB — APPROVED** [review]
lgtm

**Comment by RyanZhaoXB** [comment]
hello, @hey-kong  . your commit should be provided with sign-off information. you can try to re-commit with this command git commit -s -m "XXX" or git commit -s --amend and push your commit again. please check this https://github.com/kubeedge/mappers-go/pull/68/checks?check_run_id=6051544182

**Comment by RyanZhaoXB** [comment]
@fisherxu please help to review this pr. thanks.

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**